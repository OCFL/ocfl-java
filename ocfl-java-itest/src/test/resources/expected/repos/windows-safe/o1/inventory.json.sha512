39f91699402eba61c3032767dfb77a05f22f4ec9a645b1847c9ab672fe6b7887b5a2ef2fa8a5adbc3b60f3aefafd8a649b5f232143228fdab8d45f187016123c  inventory.json
