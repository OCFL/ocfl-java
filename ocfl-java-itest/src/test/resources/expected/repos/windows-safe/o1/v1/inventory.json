{
  "id" : "o1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v1",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "4dff4ea340f0a823f15d3f4f01ab62eae0e5da579ccb851f8db9dfe84c58b2b37b89903a740e1ee172da793a6e79d560e5f7f9bd058a12a280433ed6fa46510a" : [ "v1/content/tést/%3cbad%3e%3aPath%201/%5c%7cobj/%3f8%2a%25id/#{something}/[0]/۞.txt" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "1",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "4dff4ea340f0a823f15d3f4f01ab62eae0e5da579ccb851f8db9dfe84c58b2b37b89903a740e1ee172da793a6e79d560e5f7f9bd058a12a280433ed6fa46510a" : [ "tést/<bad>:Path 1/\\|obj/?8*%id/#{something}/[0]/۞.txt" ]
      }
    }
  }
}